1_fetch/src/my_happy_script_ms_11-12-21.R

```diff
@@ -0,0 +1,196 @@
+##########################
+# Model_RMSE data Plot   #
+#                        #
+# Author:                #     
+# Date:                  #
+##########################
+
+
+## Libs
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
+
+
+## Check that you are in project home directory - should be in project home directory
+print(getwd())
+
+## Global Variables:
+
+in_path = '1_fetch/src' 
+out_path = '1_fetch/out'
+csv_filename_output = 'model_RMSEs.csv'
+sb_id_sciencebase = '5d925066e4b0c4f70d0d0599'
```

Comment:
While R allows the use of `=` for assignment, many R programmers find it confusing. The convention in our group is to use `<-` instead. Can you please change the code throughout this file to use `<-` for assignment instead of `=`?

More generally, we officially use the [Tidyverse style guide](https://style.tidyverse.org/), and this is worth perusing. Our adherence to it is not perfect but it's a guiding light :)

Reply:
Great, yes sticking with the `<-` for assignment then.

Reply:
think i got all of them. let me know if i missed some.